onebusaway-android/src/main/java/org/onebusaway/android/ui/weather/WeatherUtils.java

```diff
@@ -39,6 +40,16 @@ public static void setWeatherTemp(TextView weatherTempTxtView, double temp) {
         weatherTempTxtView.setText(temperatureText);
     }
 
+    public static void toggleWeatherViewVisibility(boolean shouldShow, View weatherView) {
```

Comment:
I recommend always trying to figure out how to reduce the amount of nesting that occurs from control flow within a method. I would recommend writing this method like this:

* Check whether `weatherView` is null or not, and return early if it is.
* DRY up the call to `weatherView.setVisibility()` by determining visibility with a ternary expression.

```
public static void toggleWeatherViewVisibility(boolean shouldShow, View weatherView) {
  (if weatherView == null) {
    return;
  }
  
  weatherView.setVisibility(shouldShow ? View.VISIBLE : View.GONE);
}
```